pom.xml

```diff
@@ -110,10 +122,14 @@
       <optional>true</optional>
     </dependency>
     <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-step-api</artifactId>
-      <version>2.15</version>
-      <optional>true</optional>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>LATEST</version>
```

Comment:
Can you put a normal version here? I see no need to have the LATEST meta-version for this

Reply:
actually why isn't this a test dependency now?